armotypes/vulnerabilitytypes.go

```diff
@@ -170,3 +171,8 @@ type CisaKevInfo struct {
 	KnownRansomwareCampaignUse string `json:"knownRansomwareCampaignUse"`
 	Notes                      string `json:"notes"`
 }
+
+type IgnoreRuleSummary struct {
+	NumberOfSystemRules int      `json:"numberOfSystemRules"`
```

Comment:
no need for NumberOfSystemRules for vuln ignore rule

Reply:
it's the same type as the existing one in the dashboard, I just moved it to armoapi